vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -65,6 +66,11 @@ class Dropdown(VizroBaseModel):
     title: str = Field("", description="Title to be displayed")
     actions: list[Action] = []
 
+    # A private property that allows dynamically updating components
```

Comment:
Should this comment only be here, or everywhere?

Reply:
It applies everywhere but I'd remove the comment rather than adding it in other places because the name is quite self-explanatory.